--- pam-what-0.4.3.orig/src/db.h
+++ pam-what-0.4.3/src/db.h
@@ -20,7 +20,7 @@
 #include <mysql/mysql.h>
 typedef MYSQL db_conn;
 #elif USE_POSTGRES
-#include <libpq-fe.h>
+#include <postgresql/libpq-fe.h>
 typedef PGconn db_conn;
 typedef PGresult db_result;
 #endif
--- pam-what-0.4.3.orig/src/backends/postgres.c
+++ pam-what-0.4.3/src/backends/postgres.c
@@ -12,7 +12,7 @@
 #include <stdio.h>
 #include <string.h>
 #include <syslog.h>
-#include <libpq-fe.h>
+#include <postgresql/libpq-fe.h>
 
 #include "db.h"
 #include "pam_sql.h"
